include/multipass/delayed_shutdown_timer.h

```diff
@@ -47,7 +47,7 @@ class DelayedShutdownTimer : public QObject
 
     QTimer shutdown_timer;
     VirtualMachine* virtual_machine;
-    SSHSession ssh_session;
+    optional<SSHSession> ssh_session;
```

Comment:
hahaha, yeah:)  But really, it was the only way to do it without jumping through hoops of making an SSHSession default constructor with no arguments since we had to have the variable outside the try block.